src/main/java/com/fasterxml/jackson/databind/jsontype/impl/StdTypeResolverBuilder.java

```diff
@@ -6,6 +6,8 @@
 
 import com.fasterxml.jackson.databind.*;
 import com.fasterxml.jackson.databind.cfg.MapperConfig;
+import com.fasterxml.jackson.databind.introspect.AnnotatedClass;
+import com.fasterxml.jackson.databind.introspect.AnnotatedClassResolver;
 import com.fasterxml.jackson.databind.jsontype.*;
```

Comment:
Can you please use specific imports?

Reply:
Do you mean replace
```
import com.fasterxml.jackson.databind.*;
```
to
```
import com.fasterxml.jackson.databind.DeserializationConfig;
import com.fasterxml.jackson.databind.JavaType;
import com.fasterxml.jackson.databind.SerializationConfig;
```
and other imports like this?
It was original imports. I can change them if it's really needed. I just need to clarify if I understand you correctly.

Reply:
No, you do no have to do that. Not sure why @rakeshhny suggested that.

Reply:
In our project we follow some guidelines and in which having specific imports is one of them. This is why I have asked to use specific imports.

Wild card imports is a feature provided by the language and should be used whenever required.

I think it is about the standard practices we use in a project. If here we want to use wildcards for imports greater than or equal to number - n, then I am also fine with this.

I agree to @cowtowncoder that it does not matters a lot.

Reply:
@rakeshhny My surprise has more to do with your suggestion -- if you are working with submitter on source side, fine, but in general I don't want others to suggest their own preferences here. If I feel strongly about style I can do that myself. 
But in general I am proponent of "when in Rome", and it should be possible to get a feel of style by observing existing codebase.

So that's all. I know some developers are firmly against wildcarding; I am not. Conversely Jackson projects accept use of wild carding to reduce amount of boilerplate imports.